app/models/membership.rb

```diff
@@ -4,5 +4,16 @@ class Membership < ApplicationRecord
   belongs_to :organization
   belongs_to :user
 
+  STATUSES = [
+    :active,
+    :revoked,
+  ].freeze
+
+  enum status: STATUSES
   enum role: [:admin]
+
+  def mark_as_revoked!(timestamp = Time.zone.now)
```

Comment:
Is that to prevent cases where timezone is not defined ? 

I can see a lot of `Time.zone.now` usage in the app, should we create a task to update them all ?

Reply:
Asking because on the output I can see the value is the same

Reply:
Can you create unit tests for this? 😅

Reply:
`Time.current` is preferred over `Time.zone.now` 😅

Reply:
Code is updated to use `Time.current`. Also added a test for the membership model!